Codeinwp/neve
# fix: meta box sidebar icon not receiving props [closes #2389]

### Summary
Fixes issue where editor meta box icon was out of place.


### Will affect visual aspect of the product

NO

### Screenshots 
![image](https://user-images.githubusercontent.com/15010186/105495289-aa926b80-5cc4-11eb-8c0d-9d4d2419eba6.png)
![image](https://user-images.githubusercontent.com/15010186/105493887-a6654e80-5cc2-11eb-839b-f429c9fc7987.png)

### Test instructions


- add a new post or edit an existing one
- open the top right sidebar from the editor the Neve icon should be there
- it should be properly placed
- same thing should go for the white label module from the pro version


Closes #2389.
